services/post.js

```diff
@@ -1,11 +1,66 @@
 const express = require('express')
 const router = express.Router()
 const { ERROR } = require('../constant')
+const Post = require('../schemas/post')
 
 router.get('/post', (req, res) => {
   try {
   } catch (error) {
     res.status(500).send({ message: ERROR.POST })
   }
 })
+
+//create a post
+router.post('/', async (req,res) => {
+  const newPost = new Post(req.body)
+  try {
+    const savedPost = await newPost.save();
+    res.status(200).json(savedPost);
+  } catch (error) {
+    res.status(500).json(error)
+  }
+})
+
+//update a post
+router.put('/:id', async (req,res) => {
+  try{
+    const post = await Post.findById(req.params.post_id);
+    if(post.user_id === req.body.user_id){
+      await post.updateOne({$set:req.body});
+      res.status(200).json("Your post have been updated");
```

Comment:
should ne return the latest item. 
check it out [here](https://github.com/khaingmintunn/bulletin-board-api/blob/5b9c00325a5b9dbc27d7f8631a91ae5b1930a8ec/models/user.js#L100)